### PR TITLE
docs(jsdoc): update jsdoc for oauth tooling

### LINKED ISSUE #183: Update documentation oauth tooling in code
Follow up of #175

### PR DESCRIPTION
**Pull request by bzums**
close #183

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ getAccessToken(config)
 #### Arguments
 
 * [`config: OAuthConfig`](#tbd) - OAuth configuration for the request (specify grant type and corresponding parameters)
-* [`logger?: Logger`](#tbd) - Optional logger
+* [`logger?: Logger`](#tbd) - optional logger
```
Comment (ISO50) [inline API.md]: actually we decided to leave 'optional' out because we have the `?`syntax

Comment (bzums) [reply API.md]: 👍

File: src/oauth-tooling.ts [hunk 2]
```diff
@@ -142,13 +142,13 @@ function buildRequestAccessTokenUrl(accessTokenEndpoint: string, queryParams?: O
 
 /**
  * Makes a request to the `tokenInfoUrl` to validate the given `accessToken`.
- * Resolves with an object containing access token information in case of success.
+ * Resolves with an object containing token validation information in case of success.
```
Comment (ISO50) [inline src/oauth-tooling.ts]: Resolves with an token in case of success.

Comment (bzums) [reply src/oauth-tooling.ts]: 👍

**Comment by bzums** [comment]
👍

**Comment by ISO50** [comment]
👍

**Pull request merged**